Calendar/Get-CalendarDiagnosticObjectsSummary.ps1

```diff
@@ -162,14 +163,13 @@ if (-not ([string]::IsNullOrEmpty($Subject)) ) {
                     if ($LogToExamine.count -gt 100) {
                         Write-Host -ForegroundColor Cyan "`t This is a large number of logs to examine, this may take a while."
                     }
-                    Write-Host -ForegroundColor Cyan "`t Ignore the next [$($LogToExamine.count)] warnings..."
                     $logLeftCount = $LogToExamine.count
 
                     $ExceptionLogs = $LogToExamine | ForEach-Object {
                         $logLeftCount -= 1
                         Write-Verbose "Getting Exception Logs for [$($_.ItemId.ObjectId)]"
-                        Get-CalendarDiagnosticObjects -Identity $ID -ItemIds $_.ItemId.ObjectId -ShouldFetchRecurrenceExceptions $true -CustomPropertyNames $CustomPropertyNameList -ShouldBindToItem $true
-                        if ($logLeftCount % 20 -eq 0) {
+                        Get-CalendarDiagnosticObjects -Identity $ID -ItemIds $_.ItemId.ObjectId -ShouldFetchRecurrenceExceptions $true -CustomPropertyNames $CustomPropertyNameList -ShouldBindToItem $true 3>> $null
```

Comment:
What is this doing? Why are we adding `3>> $null` here?

Reply:
Okay, looks like we are just trying to get rid of the warning stream. If that is the case, we should then use `-WarningAction SilentlyContinue` to make it a little easier to read.

Reply:
Sweet, that worked.  I thought I had tried that before.